volumes/nginx/sites-enabled/collab-server

```diff
@@ -0,0 +1,22 @@
+server {
+  listen 80;
+  server_name collab.cofacts.tw;
+
+  # https managed by cloudflare
+
+  location / {
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto; # From cloudflare
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header Host $host;
+    proxy_set_header X-NginX-Proxy true;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection "upgrade";
+    proxy_http_version 1.1;
+    proxy_max_temp_file_size 0;
+    proxy_redirect off;
+    proxy_read_timeout 240s;
+
+    proxy_pass http://collab-server:5001/;
```

Comment:
This goes through docker network instead of port exposed on host machine, so it should align with server port instead

```suggestion
    proxy_pass http://collab-server:1234/;
```